packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js

```diff
@@ -324,6 +324,64 @@ describe("react-wildcat-prefetch", () => {
                 ReactTestUtils.renderIntoDocument(<WrappedPrefetch title={"Test Title"} />);
             });
 
+            it("hydrates multiple React components", (done) => {
+                let renderCount = 1;
+
+                class FirstHydrationTest extends React.Component {
+                    static propTypes = {
+                        firstData: React.PropTypes.object
+                    }
+
+                    static defaultProps = {
+                        firstData: {}
+                    }
+
+                    render() {
+                        expect(this.props.firstData)
+                            .to.eql(stubs.prefetchedData.firstData);
+
+                        if (renderCount > 1) {
+                            done();
+                        }
+
+                        renderCount++;
+                        return <div>{Object.keys(this.props)}</div>;
+                    }
+                }
+
+                class SecondHydrationTest extends React.Component {
+                    static propTypes = {
+                        secondData: React.PropTypes.object
+                    }
+
+                    static defaultProps = {
+                        secondData: {}
+                    }
+
+                    render() {
+                        expect(this.props.secondData)
+                            .to.eql(stubs.prefetchedData.secondData);
+
+                        if (renderCount > 1) {
+                            done();
+                        }
+
+                        renderCount++;
+                        return <div>{Object.keys(this.props)}</div>;
+                    }
+                }
```

Comment:
These 2 components are the same except for `props.firstData` and `props.secondData`. Maybe could just use one class and pass that as a prop? `<WrappedPrefetch title={"Test Title"} prefetchedData={"firstData"} />`. Would use the static prop validation but I'm not sure you need it here.